rust-mode.el

```diff
@@ -170,7 +170,18 @@ function or trait.  When nil, where will be aligned with fn or trait."
   "Face for the question mark operator."
   :group 'rust-mode)
 
+(defface rust-builtin-formatting-macro-face
+  '((t :inherit font-lock-builtin-face))
+  "Face for builtin formatting macros \(print! &c.\)."
```

Comment:
You don't need to escape parens in strings unless they're on the start of a line:

```emacs-lisp
"foo (bar)
\(baz) (biz)
```